Cancellation of the event: dates are removed

complexbrains opened the pull request:
Help with the merge, please!

complexbrains committed:
Cancellation of the event: dates are removed

jhlegarreta commented on _posts/2020-02-27-geneva.markdown:
```diff
@@ -1,18 +1,18 @@
 URL: [https://www.brainhack.ch](https://www.brainhack.ch)
 
-Description: Come and join our 2-day ½ Brainhack on March Friday 27th / Saturday 28th with project piched on the Thursday 26th late afternoon! While all projects will deal with the brain, quite a few will explore beyond imaging, and you can even propose your own. Everyone is welcome, whatever their background. For more information, and to join us, check out our website at [www.brainhack.ch](www.b ...
+Description: The event is aimed to be run within the 2020 but the exact dates of the event will be advertised later. This event allows projects will deal with the brain, quite a few will explore beyond imaging, and you can even propose your own. Everyone is welcome, whatever their background. For more information, and to join us, check out our website at [www.brainhack.ch](www.brainhack.ch)
```
The event is expected to run at a yet unspecified date during 2020. The exact dates will be announced through the event's website. This event will include projects that deal with a variety of neuroscience topics, not limited to brain imaging, and attendees will be able to propose their own projects. Everyone is welcome, whatever their background is.  For more (...).

jhlegarreta reviewed (commented):
Suggested changes. Otherwise, you can merge at anytime.

anibalsolon committed:
Adding real date, replacing our text w suggestions

Using @jhlegarreta suggestions for our text, but keeping the original text (from them).

anibalsolon commented:
Kept the date as a real date in the future.
Added @jhlegarreta suggestion in our text, but kept their original version.
